e-learning-webapps/src/main/webapp/vue-app/components/Themes/ThemeDashboardToolbar.vue

```diff
@@ -31,11 +31,12 @@
 
       <v-scale-transition>
         <div id="theme_filter_input">
-          <v-text-field        
+          <v-text-field
+            v-model="keyword"        
             :placeholder="$t('addon.elearning.filter.placeholder')"
             prepend-inner-icon="fa-filter"
             class="pa-0 ms-3 me-3 my-auto"
-            clearable />
+            :clearable="clearKeyword" />

```

Comment:
No need to bind this property to a function you can simply set it to 'clearable'